optimum/habana/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -403,6 +404,26 @@ def __call__(
         import habana_frameworks.torch as ht
         import habana_frameworks.torch.core as htcore
 
+        # Enable distributed CFG (classifier-free guidance) across 2 devices
+        # (conditional and unconditional parts are processed separately) if specified
+        use_distributed_cfg = kwargs.get("use_distributed_cfg", None)
+        if use_distributed_cfg:
```

Comment:
```suggestion
        if use_distributed_cfg is not None and use_distributed_cfg:
```

this is a personal choice, ignore if you do not like it

Reply:
I am thinking to make use_distributed_cfg actual function argument of boolean type to make it clearer and cleaner.  Will do in next iteration

Reply:
Updated the code, now `use_distributed_cfg` is a bool argument (None is not a valid value)